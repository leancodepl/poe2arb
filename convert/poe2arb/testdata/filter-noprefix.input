[
    {
        "term": "fooBar",
        "definition": "Content",
        "context": "",
        "term_plural": "",
        "reference": "",
        "comment": ""
    },
    {
        "term": "fooBarWithParam",
        "definition": "That's a {param}",
        "context": "",
        "term_plural": "",
        "reference": "",
        "comment": ""
    },
    {
        "term": "prefix:prefixedFooBar",
        "definition": "Other content",
        "context": "",
        "term_plural": "",
        "reference": "",
        "comment": ""
    },
    {
        "term": "prefix:prefixedFooBarWithParam",
        "definition": "This is {param}",
        "context": "",
        "term_plural": "",
        "reference": "",
        "comment": ""
    }
]
