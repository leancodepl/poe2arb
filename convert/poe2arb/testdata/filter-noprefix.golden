{
    "@@locale": "en",
    "fooBar": "Content",
    "fooBarWithParam": "That's a {param}",
    "@fooBarWithParam": {
        "placeholders": {
            "param": {
                "type": "String"
            }
        }
    }
}
